src/middlewared/middlewared/plugins/nfs.py

```diff
@@ -0,0 +1,47 @@
+from middlewared.schema import accepts, Bool, Dict, Int, IPAddr, List
+from middlewared.validators import Range
+from middlewared.service import SystemServiceService, ValidationErrors
+
+
+class NFSService(SystemServiceService):
+
+    class Config:
+        service = "nfs"
+        datastore_prefix = "nfs_srv_"
+
+    @accepts(Dict(
+        'nfs_update',
+        Int('servers', validators=[Range(min=1, max=256)]),
+        Bool('udp'),
+        Bool('allow_nonroot'),
+        Bool('v4'),
+        Bool('v4_v3owner'),
+        Bool('v4_krb'),
+        List('bindip', items=[IPAddr('ip')]),
+        Int('mountd_port', required=False, validators=[Range(min=1, max=65535)]),
+        Int('rpcstatd_port', required=False, validators=[Range(min=1, max=65535)]),
+        Int('rpclockd_port', required=False, validators=[Range(min=1, max=65535)]),
+        Bool('16'),
```

Comment:
@themylogin having an attribute called "16" is twitching my OCD, would it be hard (and agree) to change it to something more meaningful? `big_groups`, `groups_limit` or something?